Makefile

```diff
@@ -36,12 +36,12 @@ endif
 ifndef HAS_DEP
 	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 endif
-	dep ensure
```

Comment:
can you please remove the `cd $(DEST)`

Reply:
but without this change, i can see the failure when building the binary:
```shell
root@lingxian-docker-registry:~/cloud-provider-openstack# make octavia-ingress-controller
cd /home/ubuntu/go/src/k8s.io/cloud-provider-openstack
dep ensure
/home/ubuntu/cloud-provider-openstack is not within a known GOPATH/src
Makefile:39: recipe for target 'depend' failed
make: *** [depend] Error 1
```

not sure if I missed something.